python-avocado.spec

```diff
@@ -175,7 +175,10 @@ popd
 pushd optional_plugins/resultsdb
 %{__python} setup.py develop --user
 popd
-selftests/run
+# Package build environments have the least amount of resources
+# we have observed so far.  Let's avoid tests that require too
+# much resources or are time sensitive
+AVOCADO_CHECK_LEVEL=0 selftests/run
```

Comment:
This brings an chicken-egg problem... The `AVOCADO_CHECK_LEVEL` is not used, nor defined at this point in Avocado sources. I think it's better to first introduce a concept and then use it, but I know the patch adding it is later in this patch set so this note is just for your consideration, not a demand...